scripts/gtk-bundle-from-msys2.sh

```diff
@@ -157,16 +179,27 @@ extract_packages() {
 
 move_extracted_files() {
 	echo "Move extracted data to destination directory"
-	if [ -d mingw32 ]; then
+	if [ -d mingw64 ]; then
 		for d in bin etc home include lib locale share var; do
-			if [ -d "mingw32/$d" ]; then
+			if [ -d "mingw64/$d" ]; then
 				rm -rf $d
 				# prevent sporadic 'permission denied' errors on my system, not sure why they happen
 				sleep 0.5
-				mv mingw32/$d .
+				mv mingw64/$d .
 			fi
 		done
-		rmdir mingw32
+		rmdir mingw64
+	fi
+}
+
+delayed_post_install() {
+	if [ "$run_pi" ]; then
+		echo "Execute delayed post install tasks"
+		# Commands have been collected manually from the various .INSTALL scripts
```

Comment:
This seems quite fragile to me, isn't it kind of risky to have to maintain that on our own?
Also, was running individual post-install script just too slow, or other problem?  One obvious solution if speed was not the issue but e.g. dependency order or something would be to run them all after (like, here).

Reply:
Yes, it is fragile. But no more than having them executed during package extraction:
```
Running post_install script for "glib2"
.INSTALL: line 3: pwd: -W: invalid option
pwd: usage: pwd [-LP]
/build
sed: can't read /bin/gdbus-codegen-script.py: No such file or directory
sed: can't read /bin/glib-genmarshal-script.py: No such file or directory
sed: can't read /bin/glib-mkenums-script.py: No such file or directory
sed: can't read /bin/gtester-report-script.py: No such file or directory
```
They use `pwd -W` which is a MSYS special option and exist in no other `pwd` implementation.

And it continues:
```
Running post_install script for "fontconfig"

  Fontconfig configuration is done via /mingw64/etc/fonts/conf.avail and conf.d.
  Read /mingw64/etc/fonts/conf.d/README for more information.

updating font cache... .INSTALL: line 13: mingw64/bin/fc-cache: No such file or directory
done.
Running post_install script for "gdk-pixbuf2"
.INSTALL: line 2: mingw64/bin/gdk-pixbuf-query-loaders: No such file or directory
Running post_install script for "hicolor-icon-theme"
.INSTALL: line 9: mingw64/bin/gdk-pixbuf-query-loaders.exe: cannot execute binary file: Exec format error
Running post_install script for "adwaita-icon-theme"
.INSTALL: line 9: mingw64/bin/gdk-pixbuf-query-loaders.exe: cannot execute binary file: Exec format error
```
Many of the scripts don't work properly and much of the stuff in there we don't need.
We could maybe patch the install scripts so that they work with `wine` and replace `pwd -W` but this feels even more fragile.

Reply:
Ouch.  OK then, makes sense and it seems actually *more* reliable that way.